### PR TITLE
Ingest lambda timeout

### PR DESCRIPTION
**Pull request by sandyhider**

### CODE REVIEW

File: activities/tests/test_ingest_queue_populate.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# without updating scripts responsible for deploying the lambda code.
+
+import boto3
+import ingest_queue_populate as iqp
+import unittest
+from unittest.mock import patch, MagicMock
+
+@patch('boto3.resource')
+class TestIngestQueueUploadLambda(unittest.TestCase):
+    # def test_something(self, fake_resource):
```
Comment (movestill) [inline activities/tests/test_ingest_queue_populate.py]: Get rid of this commented out sample test.

File: activities/ingest_queue_populate.py [hunk 2]
```diff
@@ -79,13 +86,20 @@ def ingest_populate(args):
                      poll_delay = POLL_DELAY,
                      status_delay = STATUS_DELAY)
 
-    total_sent = reduce(lambda x, y: x+y, results, 0)
+    tile_count = get_tile_count(args)
+    messages_uploaded = reduce(lambda x, y: x+y, results, 0)
```
Comment (derek-pryor) [inline activities/ingest_queue_populate.py]: why not use `sum(results)`?

File: activities/ingest_queue_populate.py [hunk 3]
```diff
@@ -79,13 +86,20 @@ def ingest_populate(args):
...
+    if tile_count != messages_uploaded:
+        log.debug("Warning.  Messages uploaded do not match tile count.  tile count: {} messages uploaded: {}"
```
Comment (derek-pryor) [inline activities/ingest_queue_populate.py]: Why not use `log.warning()` if the message is a warning?

File: activities/ingest_queue_populate.py [hunk 4]
```diff
@@ -98,24 +112,39 @@ def clear_queue(arn):
...
 
-    for t in range_('t'):
-        for z in range_('z'):
-            args_ = args.copy()
+def get_tile_count(args):
+    tile_size = lambda v: args[v + "_tile_size"]
+    extent = lambda v: args[v + '_stop'] - args[v + '_start']
+    num_tiles_in = lambda v: math.ceil(extent(v) / tile_size(v))
+
+    x_extent = args['x_stop'] - args['x_start']
```
Comment (derek-pryor) [inline activities/ingest_queue_populate.py]: Why not use the `extent` lambda you define above?

File: activities/ingest_queue_populate.py (hunk 2 above)
Comment (sandyhider) [reply activities/ingest_queue_populate.py]: Updated.

File: activities/ingest_queue_populate.py (hunk 3 above)
Comment (sandyhider) [reply activities/ingest_queue_populate.py]: Updated.

File: activities/ingest_queue_populate.py (hunk 4 above)
Comment (sandyhider) [reply activities/ingest_queue_populate.py]: Updated.

**Pull request merged**